statistics-dashboard-common/src/main/java/statistics/dashboard/common/models/filters/StatisticsFilter.java

```diff
@@ -0,0 +1,8 @@
+package statistics.dashboard.common.models.filters;
+
+/**
+ * Interface to represent filters for Statistics Dashboard
+ */
+public interface StatisticsFilter {
```

Comment:
This empty interface is not used/referenced anywhere... is this needed? Perhaps you anticipate that it will be needed?